deploy/cron.js

```diff
@@ -37,50 +53,50 @@ var api_run = function() {
 	})
 }
 
-// var daily_run = function() {
-// 	logger.info("about to run daily.sh");
-
-// 	var daily = spawn(`${scriptRootPath}/daily.sh`)
-// 	daily.stdout.on("data", (data) => {
-// 		logger.info("[daily.sh]", data.toString().trim())
-// 	})
-// 	daily.stderr.on("data", (data) => {
-// 		logger.info("[daily.sh]", data.toString().trim())
-// 	})
-// 	daily.on("exit", (code) => {
-// 		logger.info("daily.sh exitted with code:", code)
-// 	})
-// }
-
-// var hourly_run = function(){
-// 	logger.info("about to run hourly.sh");
-
-// 	var hourly = spawn(`${scriptRootPath}/hourly.sh`)
-// 	hourly.stdout.on("data", (data) => {
-// 		logger.info("[hourly.sh]", data.toString().trim())
-// 	})
-// 	hourly.stderr.on("data", (data) => {
-// 		logger.info("[hourly.sh]", data.toString().trim())
-// 	})
-// 	hourly.on("exit", (code) => {
-// 		logger.info("hourly.sh exitted with code:", code)
-// 	})
-// }
-
-// var realtime_run = function(){
-// 	logger.info("about to run realtime.sh");
-
-// 	var realtime = spawn(`${scriptRootPath}/realtime.sh`)
-// 	realtime.stdout.on("data", (data) => {
-// 		logger.info("[realtime.sh]", data.toString().trim())
-// 	})
-// 	realtime.stderr.on("data", (data) => {
-// 		logger.info("[realtime.sh]", data.toString().trim())
-// 	})
-// 	realtime.on("exit", (code) => {
-// 		logger.info("realtime.sh exitted with code:", code)
-// 	})
-// }
+var daily_run = function() {
```

Comment:
This were commented out because we were runing the application on this branch, but we revert so that develop includes all the methods to run a full cron task sequence.